core/src/main/java/org/wso2/carbon/kernel/config/model/DeploymentConfig.java

```diff
@@ -15,17 +15,32 @@
  */
 package org.wso2.carbon.kernel.config.model;
 
+
+import org.wso2.carbon.kernel.annotations.Configuration;
+import org.wso2.carbon.kernel.annotations.Element;
+
 /**
  * Deployment Config bean.
  *
  * @since 5.0.0
  */
+@Configuration(description = "Deployment Engine related configurations")
 public class DeploymentConfig {
```

Comment:
This class is not used in kernel anymore. It is now used with carbon-deployment. So can we check and remove these references from kernel code base?